Library/Stubs/Generator.php

```diff
@@ -348,7 +348,10 @@ protected function buildMethod(ClassMethod $method, bool $isInterface, string $i
             if ($supported > 1) {
                 $return = '';
             }
+        } elseif (version_compare(PHP_VERSION, '7.1.0', '>=') && $method->isVoid()) {
```

Comment:
Better to use comparison with constant, its faster:

```php
elseif (PHP_VERSION_ID >= 70100 && $method->isVoid())
```

Reply:
@AlexNDRmac 
thank you.
I adjusted it for other parts, but is this better?

Reply:
Oh... I see :) This should be refactored... I think we can leave it as is for now. Thanks :)